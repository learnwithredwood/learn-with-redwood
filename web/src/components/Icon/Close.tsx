interface Props {
  className: string
  height: number
  width: number
}

const Close = ({
  className = '',
  height = 24,
  width = 24,
}: Props): JSX.Element => {
  return (
    <svg
      className={className}
      width={width}
      height={height}
      viewBox="0 0 24 24"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path d="M23.9998 2.41713L21.5827 0L11.9999 9.58279L2.41713 0L0 2.41713L9.58279 11.9999L0 21.5827L2.41713 23.9998L11.9999 14.417L21.5827 23.9998L23.9998 21.5827L14.417 11.9999L23.9998 2.41713Z" />
    </svg>
  )
}

export { Close }
