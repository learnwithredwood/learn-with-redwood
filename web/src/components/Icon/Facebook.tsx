interface Props {
  className: string
  height: number
  width: number
}

const Facebook = ({
  className = '',
  height = 42,
  width = 42,
}: Props): JSX.Element => {
  return (
    <svg
      className={className}
      width={width}
      height={height}
      viewBox="0 0 42 42"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path d="M16.7 22.7V37H23.7V22.8H28.5L29.7 16.9H23.8C23.8 16.9 23.8 14.1 23.8 12.5C23.7 11.9 24 11 25 11C26.5 11 28.4 11 28.4 11V5C28.4 5 25.4 5 22.1 5C19.9 5 16.7 7.7 16.7 10.5C16.7 13.4 16.7 16.6 16.7 16.6H13V22.6L16.7 22.7Z" />
    </svg>
  )
}

export { Facebook }
