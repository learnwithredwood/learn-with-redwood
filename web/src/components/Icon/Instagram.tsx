interface Props {
  className: string
  height: number
  width: number
}

const Instagram = ({
  className = '',
  height = 42,
  width = 42,
}: Props): JSX.Element => {
  return (
    <svg
      className={className}
      width={width}
      height={height}
      viewBox="0 0 42 42"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path d="M34.5 3H7.5C5.025 3 3 5.025 3 7.5V34.5C3 36.975 5.025 39 7.5 39H34.5C36.975 39 39 36.975 39 34.5V7.5C39 5.025 36.975 3 34.5 3ZM21 16.5C25.1625 16.5 25.5 21 25.5 21C25.5 21 25.6125 25.5 21 25.5C16.3875 25.5 16.5 21 16.5 21C16.5 21 16.8375 16.5 21 16.5ZM34.5 32.25C34.5 33.4875 33.4875 34.5 32.25 34.5H9.75C8.5125 34.5 7.5 33.4875 7.5 32.25V21C7.5 19.7625 8.5125 18.75 9.75 18.75H12.3375C12.1125 19.425 12 20.2125 12 21C12 25.95 16.05 30 21 30C25.95 30 30 25.95 30 21C30 20.2125 29.8875 19.425 29.6625 18.75H32.25C33.4875 18.75 34.5 19.7625 34.5 21V32.25ZM34.5 12C34.5 13.2375 33.4875 14.25 32.25 14.25H30C28.7625 14.25 27.75 13.2375 27.75 12V9.75C27.75 8.5125 28.7625 7.5 30 7.5H32.25C33.4875 7.5 34.5 8.5125 34.5 9.75V12Z" />
    </svg>
  )
}

export { Instagram }
