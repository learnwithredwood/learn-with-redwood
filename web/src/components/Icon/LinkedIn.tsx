interface Props {
  className: string
  height: number
  width: number
}

const LinkedIn = ({
  className = '',
  height = 42,
  width = 42,
}: Props): JSX.Element => {
  return (
    <svg
      className={className}
      width={width}
      height={height}
      viewBox="0 0 42 42"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path d="M12.2 8.4C12.2 10.3 10.8 11.8 8.5 11.8C6.4 11.8 5 10.3 5 8.4C5 6.5 6.4 5 8.6 5C10.9 5 12.2 6.5 12.2 8.4ZM5.2 36.6V14.6H12V36.6H5.2ZM16.1 21.6C16.1 18.9 16 16.6 15.9 14.6H21.8L22.1 17.7H22.2C23.1 16.3 25.3 14.1 29 14.1C33.5 14.1 36.9 17.1 36.9 23.6V36.6H30.1V24.4C30.1 21.6 29.1 19.6 26.6 19.6C24.7 19.6 23.6 20.9 23.1 22.2C23 22.6 23 23.2 23 23.9V36.6H16.2V21.6H16.1Z" />
    </svg>
  )
}

export { LinkedIn }
