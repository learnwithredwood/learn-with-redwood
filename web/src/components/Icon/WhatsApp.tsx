interface Props {
  className: string
  height: number
  width: number
}

const WhatsApp = ({
  className = '',
  height = 42,
  width = 42,
}: Props): JSX.Element => {
  return (
    <svg
      className={className}
      width={width}
      height={height}
      viewBox="0 0 42 42"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path d="M29.638 23.6862C29.5679 23.6525 26.9434 22.3601 26.4771 22.1923C26.2867 22.1239 26.0827 22.0572 25.8658 22.0572C25.5114 22.0572 25.2137 22.2338 24.9818 22.5807C24.7196 22.9704 23.926 23.8981 23.6807 24.1752C23.6487 24.2118 23.605 24.2555 23.5788 24.2555C23.5553 24.2555 23.1492 24.0882 23.0262 24.0348C20.2108 22.8119 18.0739 19.871 17.7808 19.375C17.739 19.3037 17.7372 19.2714 17.7369 19.2714C17.7472 19.2336 17.8418 19.1387 17.8907 19.0897C18.0337 18.9483 18.1885 18.7619 18.3384 18.5815C18.4093 18.4961 18.4804 18.4105 18.5502 18.3299C18.7676 18.0769 18.8643 17.8806 18.9766 17.6531L19.0353 17.5349C19.3093 16.9906 19.0753 16.5313 18.9997 16.3829C18.9376 16.2588 17.8295 13.5843 17.7116 13.3033C17.4283 12.6253 17.0539 12.3096 16.5337 12.3096C16.4854 12.3096 16.5337 12.3096 16.3313 12.3181C16.0847 12.3285 14.7423 12.5052 14.1488 12.8794C13.5194 13.2762 12.4546 14.5411 12.4546 16.7656C12.4546 18.7677 13.7251 20.6581 14.2706 21.377C14.2842 21.3951 14.3091 21.4319 14.3452 21.4848C16.4342 24.5357 19.0385 26.7966 21.6785 27.8513C24.2202 28.8665 25.4237 28.9839 26.1079 28.9839C26.1081 28.9839 26.1081 28.9839 26.1081 28.9839C26.3956 28.9839 26.6257 28.9613 26.8288 28.9413L26.9576 28.9291C27.8356 28.8512 29.7651 27.8514 30.204 26.6317C30.5497 25.6711 30.6409 24.6215 30.4109 24.2405C30.2533 23.9815 29.9818 23.8512 29.638 23.6862Z" />
      <path d="M21.3195 3C11.5696 3 3.63745 10.8725 3.63745 20.5492C3.63745 23.679 4.47504 26.7426 6.06173 29.4239L3.02475 38.3824C2.96818 38.5494 3.01026 38.7341 3.1338 38.8599C3.22299 38.9509 3.34396 39 3.46751 39C3.51484 39 3.56253 38.9929 3.60905 38.978L12.9503 36.0097C15.5066 37.3754 18.3957 38.0964 21.3196 38.0964C31.0686 38.0965 39 30.2248 39 20.5492C39 10.8725 31.0686 3 21.3195 3ZM21.3195 34.4409C18.5682 34.4409 15.9034 33.6464 13.6127 32.1433C13.5357 32.0927 13.4463 32.0667 13.3563 32.0667C13.3087 32.0667 13.261 32.074 13.2146 32.0887L8.53521 33.5762L10.0458 29.1195C10.0947 28.9753 10.0702 28.8162 9.98023 28.6932C8.23587 26.3098 7.31378 23.4937 7.31378 20.5492C7.31378 12.8883 13.5967 6.65564 21.3194 6.65564C29.0412 6.65564 35.3234 12.8883 35.3234 20.5492C35.3235 28.2091 29.0414 34.4409 21.3195 34.4409Z" />
    </svg>
  )
}

export { WhatsApp }
