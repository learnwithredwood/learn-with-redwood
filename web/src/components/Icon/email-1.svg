<svg width="42" height="42" viewBox="0 0 42 42" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect width="42" height="42" fill="white"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M12.7016 6.75H29.299C30.5736 6.74998 31.6255 6.74996 32.4824 6.81997C33.3723 6.89269 34.1905 7.04875 34.9589 7.44029C36.1506 8.04749 37.1195 9.01637 37.7267 10.2081C38.1045 10.9496 38.2631 11.7374 38.339 12.5914C38.4237 12.8501 38.4385 13.123 38.3882 13.3835C38.417 14.0962 38.417 14.9206 38.417 15.8679V26.1321C38.417 27.4066 38.417 28.4585 38.347 29.3154C38.2743 30.2054 38.1182 31.0235 37.7267 31.7919C37.1195 32.9836 36.1506 33.9525 34.9589 34.5597C34.1905 34.9513 33.3723 35.1073 32.4824 35.18C31.6255 35.25 30.5736 35.25 29.2991 35.25H12.7015C11.427 35.25 10.3751 35.25 9.51824 35.18C8.62825 35.1073 7.81014 34.9513 7.04169 34.5597C5.85 33.9525 4.88112 32.9836 4.27392 31.7919C3.88238 31.0235 3.72632 30.2054 3.65361 29.3154C3.58359 28.4585 3.58361 27.4066 3.58363 26.1321V15.8679C3.58362 14.9206 3.5836 14.0962 3.61235 13.3835C3.56207 13.123 3.57691 12.8501 3.66156 12.5914C3.73754 11.7374 3.89609 10.9496 4.27392 10.2081C4.88112 9.01637 5.85 8.04749 7.04169 7.44029C7.81014 7.04875 8.62825 6.89269 9.51824 6.81997C10.3751 6.74996 11.427 6.74998 12.7016 6.75ZM6.7503 16.1244V26.0667C6.7503 27.4229 6.75153 28.3449 6.80976 29.0575C6.86647 29.7517 6.96927 30.1067 7.09544 30.3543C7.39904 30.9502 7.88348 31.4346 8.47933 31.7382C8.72696 31.8644 9.08195 31.9672 9.77611 32.0239C10.4888 32.0821 11.4107 32.0833 12.767 32.0833H29.2336C30.5899 32.0833 31.5118 32.0821 32.2245 32.0239C32.9186 31.9672 33.2736 31.8644 33.5213 31.7382C34.1171 31.4346 34.6016 30.9501 34.9052 30.3543C35.0313 30.1067 35.1341 29.7517 35.1908 29.0575C35.2491 28.3449 35.2503 27.4229 35.2503 26.0667V16.1244L24.8138 23.4299C24.7524 23.4729 24.6915 23.5156 24.6312 23.558C23.7688 24.1635 23.0116 24.6953 22.1492 24.9102C21.3948 25.0983 20.6058 25.0983 19.8514 24.9102C18.989 24.6953 18.2318 24.1635 17.3694 23.558C17.3091 23.5156 17.2482 23.4729 17.1868 23.4299L6.7503 16.1244ZM35.1192 12.3507L22.9979 20.8357C21.8437 21.6436 21.5992 21.7838 21.3833 21.8376C21.1318 21.9003 20.8688 21.9003 20.6173 21.8376C20.4014 21.7838 20.1569 21.6436 19.0027 20.8357L6.88135 12.3507C6.93791 12.0227 7.01151 11.8104 7.09544 11.6457C7.39904 11.0499 7.88348 10.5654 8.47933 10.2618C8.72696 10.1356 9.08195 10.0328 9.77611 9.97612C10.4888 9.9179 11.4107 9.91667 12.767 9.91667H29.2336C30.5899 9.91667 31.5118 9.9179 32.2245 9.97612C32.9186 10.0328 33.2736 10.1356 33.5213 10.2618C34.1171 10.5654 34.6016 11.0499 34.9052 11.6457C34.9891 11.8104 35.0627 12.0227 35.1192 12.3507Z" fill="black"/>
</svg>
