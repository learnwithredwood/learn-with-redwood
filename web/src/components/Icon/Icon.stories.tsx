import { Icon, Props as IconProps } from './Icon'

const Template = (args: IconProps) => <Icon {...args} />

export const Close = Template.bind({})
Close.args = {
  name: 'close',
}

export const GitHub = Template.bind({})
GitHub.args = {
  name: 'github',
}

export const Medium = Template.bind({})
Medium.args = {
  name: 'medium',
}

export const Twitter = Template.bind({})
Twitter.args = {
  name: 'twitter',
}

export const YouTube = Template.bind({})
YouTube.args = {
  name: 'youtube ',
}

export default { title: 'Components/Icon' }
