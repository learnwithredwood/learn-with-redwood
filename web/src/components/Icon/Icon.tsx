import { Close } from './Close'
import { GitHub } from './GitHub'
import { Medium } from './Medium'
import { Twitter } from './Twitter'
import { YouTube } from './YouTube'

interface Props {
  name: string
  className?: string
  width?: number
  height?: number
}

const Icon = ({ name, className, width, height }: Props): JSX.Element => {
  switch (name.toLowerCase()) {
    case 'close':
      return <Close className={className} width={width} height={height} />
    case 'github':
      return <GitHub className={className} width={width} height={height} />
    case 'medium':
      return <Medium className={className} width={width} height={height} />
    case 'twitter':
      return <Twitter className={className} width={width} height={height} />
    case 'youtube':
      return <YouTube className={className} width={width} height={height} />
    default:
      return <span />
  }
}

export default Icon
