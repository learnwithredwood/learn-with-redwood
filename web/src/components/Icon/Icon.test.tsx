import { render } from '@redwoodjs/testing/web'
import renderer from 'react-test-renderer'

import { Icon } from './Icon'

describe('Icon', () => {
  it('renders successfully', () => {
    expect(() => {
      render(<Icon name="github" />)
    }).not.toThrow()
  })

  it('renders the Close icon', () => {
    const closeIcon = renderer.create(<Icon name="close" />).toJSON()
    expect(closeIcon).toMatchSnapshot()
  })

  it('renders the Email icon', () => {
    const emailIcon = renderer.create(<Icon name="email" />).toJSON()
    expect(emailIcon).toMatchSnapshot()
  })

  it('renders the Facebook icon', () => {
    const facebookIcon = renderer.create(<Icon name="facebook" />).toJSON()
    expect(facebookIcon).toMatchSnapshot()
  })

  it('renders GitHub icon', () => {
    const gitHubIcon = renderer.create(<Icon name="github" />).toJSON()
    expect(gitHubIcon).toMatchSnapshot()
  })

  it('renders Instagram icon', () => {
    const instagramIcon = renderer.create(<Icon name="instagram" />).toJSON()
    expect(instagramIcon).toMatchSnapshot()
  })

  it('renders LinkedIn icon', () => {
    const linkedInIcon = renderer.create(<Icon name="linkedin" />).toJSON()
    expect(linkedInIcon).toMatchSnapshot()
  })

  it('renders Medium icon', () => {
    const MediumIcon = renderer.create(<Icon name="medium" />).toJSON()
    expect(MediumIcon).toMatchSnapshot()
  })

  it('renders Twitter icon', () => {
    const TwitterIcon = renderer.create(<Icon name="twitter" />).toJSON()
    expect(TwitterIcon).toMatchSnapshot()
  })

  it('renders WhatsApp icon', () => {
    const whatsAppIcon = renderer.create(<Icon name="whatsapp" />).toJSON()
    expect(whatsAppIcon).toMatchSnapshot()
  })

  it('renders YouTube icon', () => {
    const YouTubeIcon = renderer.create(<Icon name="medium" />).toJSON()
    expect(YouTubeIcon).toMatchSnapshot()
  })

  it('renders Zeal icon', () => {
    const zealIcon = renderer.create(<Icon name="zeal" />).toJSON()
    expect(zealIcon).toMatchSnapshot()
  })
})
