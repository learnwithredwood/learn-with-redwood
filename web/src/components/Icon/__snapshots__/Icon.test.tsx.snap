// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon renders GitHub icon 1`] = `
<svg
  className=""
  fill="none"
  height={42}
  viewBox="0 0 42 42"
  width={42}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clipRule="evenodd"
    d="M21.0005 0.875C16.0151 0.877587 11.1932 2.6433 7.39702 5.85642C3.60081 9.06953 1.07783 13.5205 0.279203 18.4135C-0.519425 23.3065 0.458372 28.3224 3.03777 32.5643C5.61718 36.8062 9.62997 39.9976 14.3586 41.5677C15.4018 41.7613 15.7948 41.1148 15.7948 40.565C15.7948 40.0152 15.7739 38.4213 15.767 36.6786C9.92494 37.9407 8.69047 34.2134 8.69047 34.2134C7.73767 31.793 6.36061 31.1568 6.36061 31.1568C4.455 29.8637 6.5032 29.8879 6.5032 29.8879C8.61398 30.0365 9.72327 32.042 9.72327 32.042C11.5941 35.2333 14.6368 34.3102 15.833 33.7708C16.0208 32.4188 16.5668 31.4991 17.1684 30.977C12.5017 30.4515 7.59857 28.6604 7.59857 20.6595C7.56964 18.5845 8.34404 16.5778 9.76152 15.0547C9.54592 14.5291 8.82609 12.4062 9.96667 9.5225C9.96667 9.5225 11.7297 8.96237 15.7426 11.6628C19.1847 10.7268 22.8163 10.7268 26.2583 11.6628C30.2677 8.96237 32.0273 9.5225 32.0273 9.5225C33.1713 12.3992 32.4515 14.5222 32.2359 15.0547C33.6579 16.5781 34.4339 18.5882 34.4023 20.6664C34.4023 28.6846 29.4888 30.4515 24.8152 30.9667C25.5663 31.6167 26.2374 32.8856 26.2374 34.8357C26.2374 37.6295 26.2131 39.8769 26.2131 40.565C26.2131 41.1217 26.5921 41.7717 27.6562 41.5677C32.3854 39.9974 36.3986 36.8055 38.978 32.5628C41.5573 28.3202 42.5346 23.3035 41.7351 18.4101C40.9355 13.5167 38.4113 9.06562 34.6138 5.85306C30.8164 2.6405 25.9934 0.875947 21.0074 0.875H21.0005Z"
    fillRule="evenodd"
  />
  <path
    d="M7.95276 30.8561C7.90755 30.9598 7.74064 30.9909 7.60502 30.9183C7.4694 30.8457 7.36857 30.7109 7.41725 30.6037C7.46593 30.4965 7.62937 30.4688 7.76499 30.5415C7.90061 30.6141 8.00492 30.7524 7.95276 30.8561Z"
  />
  <path
    d="M8.80475 31.8C8.73274 31.8361 8.65035 31.8461 8.57169 31.8285C8.49303 31.8108 8.423 31.7665 8.37356 31.7032C8.23794 31.558 8.21011 31.3574 8.31443 31.2675C8.41875 31.1776 8.60654 31.2191 8.74216 31.3643C8.87778 31.5096 8.90908 31.7101 8.80475 31.8Z"
  />
  <path
    d="M9.63243 32.9996C9.50377 33.0895 9.28468 32.9996 9.16297 32.8198C9.12932 32.7876 9.10256 32.7489 9.08427 32.7061C9.06599 32.6634 9.05655 32.6174 9.05655 32.5709C9.05655 32.5244 9.06599 32.4784 9.08427 32.4356C9.10256 32.3929 9.12932 32.3542 9.16297 32.3219C9.29164 32.2355 9.51072 32.3219 9.63243 32.4983C9.75414 32.6746 9.75762 32.9097 9.63243 32.9996Z"
  />
  <path
    d="M10.7556 34.1612C10.6408 34.2891 10.4079 34.2545 10.2166 34.0817C10.0253 33.9088 9.98013 33.6737 10.0949 33.5492C10.2096 33.4247 10.4426 33.4593 10.6408 33.6287C10.839 33.7981 10.8773 34.0367 10.7556 34.1612Z"
  />
  <path
    d="M12.3308 34.8392C12.2786 35.0017 12.0422 35.0743 11.8057 35.0052C11.5693 34.936 11.4128 34.7424 11.458 34.5764C11.5032 34.4104 11.7431 34.3344 11.9831 34.4104C12.223 34.4865 12.376 34.6698 12.3308 34.8392Z"
  />
  <path
    d="M14.0487 34.9567C14.0487 35.1261 13.854 35.2713 13.6036 35.2748C13.3532 35.2782 13.1481 35.1399 13.1481 34.9705C13.1481 34.8011 13.3428 34.6559 13.5932 34.6524C13.8435 34.649 14.0487 34.7838 14.0487 34.9567Z"
  />
  <path
    d="M15.6483 34.6904C15.6796 34.8598 15.5057 35.0361 15.2554 35.0776C15.005 35.1191 14.7859 35.0188 14.7546 34.8529C14.7233 34.6869 14.9042 34.5071 15.1476 34.4622C15.391 34.4172 15.617 34.5209 15.6483 34.6904Z"
  />
</svg>
`;

exports[`Icon renders Medium icon 1`] = `
<svg
  className=""
  fill="none"
  height={42}
  viewBox="0 0 42 42"
  width={42}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M23.0851 21.0002C23.0851 27.599 18.9749 32.9483 13.905 32.9483C8.83511 32.9483 4.72461 27.5978 4.72461 21.0002C4.72461 14.4027 8.8348 9.05176 13.905 9.05176C18.9752 9.05176 23.0851 14.4015 23.0851 21.0002Z"
  />
  <path
    d="M33.156 21.0002C33.156 27.2115 31.1009 32.2487 28.5658 32.2487C26.0307 32.2487 23.9756 27.2115 23.9756 21.0002C23.9756 14.7888 26.0304 9.75159 28.5655 9.75159C31.1006 9.75159 33.1557 14.7872 33.1557 21.0002"
  />
  <path
    d="M37.2753 21.0002C37.2753 26.564 36.5527 31.0772 35.6609 31.0772C34.7692 31.0772 34.0469 26.5652 34.0469 21.0002C34.0469 15.4352 34.7695 10.9233 35.6609 10.9233C36.5524 10.9233 37.2753 15.4348 37.2753 21.0002Z"
  />
</svg>
`;

exports[`Icon renders Twitter icon 1`] = `
<svg
  className=""
  fill="none"
  height={42}
  viewBox="0 0 42 42"
  width={42}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M13.2128 38.0623C29.0579 38.0623 37.727 24.9315 37.727 13.5481C37.727 13.1789 37.7188 12.8016 37.7024 12.4324C39.3888 11.2129 40.8441 9.70226 42 7.97159C40.4294 8.67037 38.7619 9.12674 37.0543 9.3251C38.8522 8.24741 40.1984 6.55439 40.8434 4.55991C39.152 5.56229 37.3023 6.26939 35.3735 6.65089C34.074 5.27005 32.3557 4.35577 30.4844 4.04941C28.6132 3.74304 26.6931 4.06166 25.021 4.95599C23.349 5.85032 22.0182 7.27055 21.2342 8.99711C20.4503 10.7237 20.257 12.6604 20.6842 14.5078C17.2594 14.336 13.9089 13.4463 10.85 11.8965C7.79105 10.3467 5.09196 8.17133 2.9277 5.51147C1.8277 7.40799 1.4911 9.6522 1.9863 11.788C2.48151 13.9238 3.77136 15.7908 5.59371 17.0098C4.2256 16.9664 2.88746 16.598 1.68984 15.9352V16.0418C1.68862 18.0321 2.37667 19.9613 3.63704 21.5017C4.89741 23.042 6.65232 24.0983 8.60344 24.491C7.3361 24.8378 6.00597 24.8883 4.71598 24.6387C5.26655 26.3503 6.33775 27.8474 7.7801 28.9209C9.22244 29.9944 10.964 30.5909 12.7616 30.627C9.70974 33.0243 5.93981 34.3245 2.05898 34.3184C1.37075 34.3173 0.683198 34.2751 0 34.1921C3.9425 36.7214 8.52868 38.0647 13.2128 38.0623Z"
  />
</svg>
`;

exports[`Icon renders YouTube icon 1`] = `
<svg
  className=""
  fill="none"
  height={42}
  viewBox="0 0 42 42"
  width={42}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M23.0851 21.0002C23.0851 27.599 18.9749 32.9483 13.905 32.9483C8.83511 32.9483 4.72461 27.5978 4.72461 21.0002C4.72461 14.4027 8.8348 9.05176 13.905 9.05176C18.9752 9.05176 23.0851 14.4015 23.0851 21.0002Z"
  />
  <path
    d="M33.156 21.0002C33.156 27.2115 31.1009 32.2487 28.5658 32.2487C26.0307 32.2487 23.9756 27.2115 23.9756 21.0002C23.9756 14.7888 26.0304 9.75159 28.5655 9.75159C31.1006 9.75159 33.1557 14.7872 33.1557 21.0002"
  />
  <path
    d="M37.2753 21.0002C37.2753 26.564 36.5527 31.0772 35.6609 31.0772C34.7692 31.0772 34.0469 26.5652 34.0469 21.0002C34.0469 15.4352 34.7695 10.9233 35.6609 10.9233C36.5524 10.9233 37.2753 15.4348 37.2753 21.0002Z"
  />
</svg>
`;

exports[`Icon renders the Close icon 1`] = `
<svg
  className=""
  fill="none"
  height={24}
  viewBox="0 0 24 24"
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M23.9998 2.41713L21.5827 0L11.9999 9.58279L2.41713 0L0 2.41713L9.58279 11.9999L0 21.5827L2.41713 23.9998L11.9999 14.417L21.5827 23.9998L23.9998 21.5827L14.417 11.9999L23.9998 2.41713Z"
  />
</svg>
`;
